### DISCUSSION TITLE
should we care about the $HOME/.hauler dir in the install script?

### DISCUSSION THREAD

**Discussion by dweomer** [hauler-dev/hauler#364] [General] [answered]
https://github.com/hauler-dev/hauler/pull/354#discussion_r1842730049:
![image](https://github.com/user-attachments/assets/c03fc467-5457-4b05-8cfd-72f16a28af89)

**Comment by dweomer** [comment]
I am probably biased because I want to phase it out but I do not think we should concern ourselves with `$HOME/.hauler` in the install script

**Comment by zackbradys** [comment, marked as answer]
From Discussion https://github.com/hauler-dev/hauler/pull/354#discussion_r1843116543...

it outputs the artifacts from the `install.sh`. no reason it is in `$HOME/.hauler` except for the fact that is what we have been using for the known location of `hauler` bits...
